conversation.py

```diff
@@ -224,3 +225,32 @@ def summarise_triage(triage_resp):
         print('Teleconsultation applicable: {}'
               .format(teleconsultation_applicable))
     print()
+
+
+def extract_intentions(text, mapping=constants.ANSWER_NORM):
```

Comment:
extract **intents** would be a better name, I think.
There's a convention to use the word “intent” in this sense, for instance:
* https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstarts/intent-recognition?pivots=programming-language-csharp
* https://towardsdatascience.com/a-brief-introduction-to-intent-classification-96fda6b1f557